### PR TITLE
DEVCTR-121: Genesys Cloud Rebranding

### PR DESCRIPTION
**Pull request by jakejrichards**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,9 +34,9 @@
     "build"
   ],
   "keywords": [
```
Comment (jakejrichards) [inline package.json:36]: Not sure if we should keep the old keywords as well for search purposes, figured they'd be okay to drop, but can add them back in if needed.

File: examples/urlInterpolationDemo.html [hunk 2]
```diff
@@ -42,21 +42,21 @@ <h2>Localization Demo</h2>
                 var DEFAULT_LANG_TAG = 'en-us';
                 var langTag = null;
                 var welcomeTokens = {
```
Comment (jakejrichards) [inline examples/urlInterpolationDemo.html:44]: Not sure if these were localized by the localization team, but figured this change was still reasonable.

Comment (justinaray) [reply examples/urlInterpolationDemo.html:44]: Yeah, I think I picked these up from a bundle somewhere just to illustrate.  Should be good to change here.

File: package.json (hunk 1 above)
Comment (justinaray) [reply package.json:36]: Hmm ... maybe keep the MyPureCloud since that's still the organization name for the time being.

Comment (justinaray) [reply package.json:36]: so, mypurecloud, genesys cloud, genesys cloud apps

**Pull request closed without merging**